tap_linkedin_ads/__init__.py

```diff
@@ -18,10 +18,11 @@
     'accounts'
 ]
 
-def do_discover(config):
+def do_discover(client, config):
 
     LOGGER.info('Starting discover')
-    check_accounts_list(config)
+    check_accounts_list(config) # Check that accounts are valid numbers.
+    client.check_accounts(config) # Check that accounts are valid LinkedIn Ads account.
```

Comment:
@savan-chovatiya There should be a single function for checking the accounts list. Can you make changes to the check_accounts_list() function to also check if the account is valid or not?

Reply:
@karanpanchal-crest I have removed check_accounts_list() function and handled that scenario in check_accounts() function itself.